### PR TITLE
Implement config.ini for enhanced persistence of settings

### PR DESCRIPTION
**Pull request by Subhanshu0027**

### CODE REVIEW

File: webui.py [hunk 1]
```diff
@@ -117,19 +128,24 @@ def removeModelCache():
     def updateExecutionProvider(provider):
         if provider == "cuda":
             if torch.cuda.is_available():
+                device = "cuda"
+                update_config(config, execution_provider=device)
```
Comment (ashish-aesthisia) [inline webui.py:132]: Remove duplicates, declare at the beginning or end

File: webui.py [hunk 2]
```diff
@@ -7,13 +7,24 @@
...
-from core import list_download_models, format_model_name, remove_dir, default_repo_id
+from core import list_download_models, remove_dir, default_repo_id, read_config, update_config
 
 cache_dir = os.path.join(os.getcwd(), "models")
 saved_models_list = list_download_models(cache_dir)
 
 #check if cuda is available
 device = 'cuda' if torch.cuda.is_available() else 'cpu'
+state, config = read_config()
+print(state,config)
```
Comment (ashish-aesthisia) [inline webui.py:18]: Remove logging

**Review by ashish-aesthisia — CHANGES_REQUESTED** [review]
Please review the requested changes.

File: webui.py (hunk 1 above)
Comment (Subhanshu0027) [reply webui.py]: Removed duplicates, declared them at the end

**Review by ashish-aesthisia — APPROVED** [review]
LGTM

**Pull request merged**